yarn.lock

```diff
@@ -1092,7 +1092,13 @@ debug@0.7.4:
   version "0.7.4"
```

Comment:
You mean make the github hide the contents? I'm not sure. It shouldn't be edited by hand. So looks like the only way is to ask GH to update their library to detect it and hide the diff by default.